tests/test_mail_parser.py

```diff
@@ -99,10 +146,6 @@ def test_issue_received(self):
             self.assertIn("date_utc", i)
             self.assertIsNotNone(i["date_utc"])
 
-        mail = mailparser.parse_from_file(mail_test_10)
```

Comment:
not sure why this is here. this case is invalid, as it has receiveds with two froms, like

```
Received: from localhost
	by smtp.notes.na.collabserv.com with smtp.notes.na.collabserv.com ESMTP
	for <lina.vailati@dududu.it> from <nicoletta_bernasconi@it.ibm.com>;
	Wed, 8 Mar 2017 16:46:15 -0000
```

it is properly returning a 'raw' key, but won't have a parsed date.